### ISSUE TITLE
NVDA cannot generate Simplified Chinese emoticon dictionary using the scons command

### ISSUE THREAD

**Issue by dingpengyu** [nvaccess/nvda#8799]
Hello, I am a translator and maintainer of NVDA Simplified Chinese. I found that I can't generate a simplified Chinese emoticon dictionary using the (scons) command. In the Chinese language system, it is divided into Simplified Chinese and Traditional Chinese. The IE8N code in Simplified Chinese is (zh_CN, or zh), while the IE8N code in Traditional Chinese is (ZH_TW, or zh_Hant). Simplified Chinese is mainly used in mainland China, while Traditional Chinese is mainly used in Taiwan. I am using Simplified Chinese, and the IE8N code is also used (zh_CN). Based on the above question, I now want to use Scons to generate a simplified Chinese emoticon dictionary, and the IE8N code is (zh_CN). However, I did not see the IE8N information in Simplified Chinese in the (cldrDict_sconscript) file. And in the process of compiling the entire NVDA, it will be stuck in the step of embedding the expression. What I am troubled now is how to solve the problem that I can't generate a simplified Chinese emoticon dictionary? Thank

**Comment by leonardder** [comment]
NVDA's locale folder contains both a zh and a zh-cn folder. Zh only seems to contain character descriptions and symbols. Are these still used?
Also, how about zh-hk?

**Comment by dingpengyu** [comment]
Zh, now it is not under maintenance, now the main maintenance is: zh_CN, zh_HK, zh_TW.

**Comment by dingpengyu** [comment]
Regarding zh_HK, it is not in my maintenance scope, I only maintain the NVDA translation in Simplified Chinese, that is (zh_CN)

**Comment by dingpengyu** [comment]
zh_HK, and zh_TW respectively have NVDA Taiwan community, NVDA Hong Kong community maintenance.

**Comment by josephsl** [comment]
Hi, caused by XML file names. Pull request on its way from my side. Thanks. CC @Nishimotz, @larry801

**Comment by leonardder** [comment]
@josephsl: could you also have a run over the other languages, including those that are commented out? I also noticed that the Korean dictionary isn't automatically picked up when espeak is set to Korean. It might make sense to create emoji dictionary for every single language that is available in the annotations list, even those that don't have an NVDA language.

**Comment by josephsl** [comment]
Hi, I think this may have to do with NVDA’s UI language more than Espeak voice language, in that UI language should override synthesizer languages unless trust voice data checkbox is on. Thanks.

**Comment by josephsl** [comment]
Hi, and so am I, and Espeak did read Korean descriptions for emojis. Thanks.
